tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "files": [],
+  "compilerOptions": {
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "resolveJsonModule": true,
+    "isolatedModules": false,
+    "target": "ES2020"
+  },
+  "references": [{ "path": "./tsconfig.app.json" }, { "path": "./tsconfig.node.json" }],
+  "include": ["eslint.config.ts", "src"]
```

Comment:
since you have config files as well in ts, so better to not restrict ts check for src folder only